### PR TITLE
Formatting and cleanup

### PR DESCRIPTION
**Pull request by bean5**
Some of this is opinionated formatting from myself, but some of it comes from the dart formatter.

I advise reviewing on per-commit basis while ignoring whitespace changes. If you do not know how to do that, feel free to let me know.

### CODE REVIEW

**Comment by bean5** [comment]
@edufolly This isn't high priority, but follows standards better.

File: example/lib/BluetoothDeviceListEntry.dart [hunk 1]
```diff
@@ -2,46 +2,66 @@ import 'package:flutter/material.dart';
...
-    title: Text(device.name ?? "Unknown device"),
-    subtitle: Text(device.address.toString()),
-    trailing: Row(
-      mainAxisSize: MainAxisSize.min,
-      children: <Widget>[
-        rssi != null ? Container(
-          margin: new EdgeInsets.all(8.0),
-          child: DefaultTextStyle(
-            style: () {
-              /**/ if (rssi >= -35) return TextStyle(color: Colors.greenAccent[700]);
```
Comment (PsychoXIVI) [inline example/lib/BluetoothDeviceListEntry.dart]: Oh, my lovely symmetry ;D

**Comment by bean5** [comment]
Did I break the symmetry? I felt that having the function was worth the
refactoring. I can revert that portion of you prefer.

**Comment by PsychoXIVI** [comment]
I just don't follow Dart code style conventions, I find all those strict rules troublesome. For example this 
```dart
              else if (rssi >= -45) return TextStyle(color: Color.lerp(Colors.greenAccent[700], Colors.lightGreen,        -(rssi + 35) / 10));
              else if (rssi >= -55) return TextStyle(color: Color.lerp(Colors.lightGreen,       Colors.lime[600],         -(rssi + 45) / 10));
              else if (rssi >= -65) return TextStyle(color: Color.lerp(Colors.lime[600],        Colors.amber,             -(rssi + 55) / 10));
              else if (rssi >= -75) return TextStyle(color: Color.lerp(Colors.amber,            Colors.deepOrangeAccent,  -(rssi + 65) / 10));
              else if (rssi >= -85) return TextStyle(color: Color.lerp(Colors.deepOrangeAccent, Colors.redAccent,         -(rssi + 75) / 10));
```
is easier to read than:
```dart
                          else if (rssi >= -45)
                            return TextStyle(
                                color: Color.lerp(Colors.greenAccent[700],
                                    Colors.lightGreen, -(rssi + 35) / 10));
                          else if (rssi >= -55)
                            return TextStyle(
                                color: Color.lerp(Colors.lightGreen,
                                    Colors.lime[600], -(rssi + 45) / 10));
                          else if (rssi >= -65)
                            return TextStyle(
                                color: Color.lerp(Colors.lime[600],
                                    Colors.amber, -(rssi + 55) / 10));
                          else if (rssi >= -75)
                            return TextStyle(
                                color: Color.lerp(
                                    Colors.amber,
                                    Colors.deepOrangeAccent,
                                    -(rssi + 65) / 10));
                          else if (rssi >= -85)
                            return TextStyle(
                                color: Color.lerp(Colors.deepOrangeAccent,
                                    Colors.redAccent, -(rssi + 75) / 10));
```
Also, I think it is stupid to have lines truncated at 80, like:
```dart
      return (sample.timestamp.millisecondsSinceEpoch - argumentsShift)
          .toDouble();
```
And, finally, mixing 2 spaces and 4 spaces indent make me sick somethimes
```dart
          trailing: Row(mainAxisSize: MainAxisSize.min, children: <Widget>[ 
            rssi != null // 2 spaces
                ? Container( // 4 spaces
                    margin: new EdgeInsets.all(8.0), // 4 spaces
                    child: DefaultTextStyle( 
                        style: () { // 4 spaces
                          /**/ if (rssi >= -35) // Why this comment even remained??  // 2 spaces
                            return TextStyle(color: Colors.greenAccent[700]); // 2 spaces
                          else if (rssi >= -45) 
                            return TextStyle( // 2 spaces
                                color: Color.lerp(Colors.greenAccent[700], // 4 spaces
                                    Colors.lightGreen, -(rssi + 35) / 10)); // 4 spaces
```

In some places it does help, in some makes it worse (at least IMO). Well, AFAIK there is no way to make `flutter format` "ignore" fragment of code (if you found some, please let me know). Afer further changes, running the `flutter format` once more also changes few things...

If you really need those formatting to be applied, let me know, I think I can accept (no changes required I think). I don't feel those, but I can understand some want the packages to follow struct Dart styling rules...

**Comment by bean5** [comment]
I don't like the autoformatter either. Arbitrarily inserting a newline statement because you use descriptive naming is annoying.

However, trailing comma is a feature that I think should actually be automatically added. That and support for leading commas. But I digress.

Up to you whether you merge. I advise it so next time I go editing files my IDE's autochanges don't get added into the mix because it won't have any.

**Pull request merged**